backend/app/mcp_server/tool_registry.py

```
@@ -0,0 +1,289 @@
+            is_async=is_async,
+        )
+
+    return sync_tool_wrapper
+
+
+def _invoke_endpoint(
+    original_func: Callable,
+    kwargs: Dict[str, Any],
+    param_names: List[str],
+    tool_name: str,
+    server_name: str,
```

Ruff (0.14.14)

[warning] Unused function argument: `server_name`

(ARG001)